wca-regulations.md

```diff
@@ -392,7 +392,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A4d3) If a stopwatch is in use, the judge starts the stopwatch as soon as the competitor starts the solve.
     - A4e) Time penalties for starting the solve are cumulative.
 - A5) During the solve:
-    - A5a) While inspecting or solving the puzzle, the competitor must not communicate with anyone other than the judge. Penalty: disqualification of the attempt (DNF).
+    - A5a) While inspecting or solving the puzzle, the competitor must not communicate with anyone other than the judge or the WCA Delegate. Penalty: disqualification of the attempt (DNF).
```

Comment:
nit: Competitions sometimes have multiple delegates listed as delegates for the competition (so referring to "the WCA delegate" can be nonsensical), and they also have delegates present who are not listed as delegates for the competition. A possible rephrasing that I'd be okay with would be "**a** WCA Delegate".

Reply:
Thanks again!
It's fixed now.